src/devices/xiaomi.ts

```diff
@@ -3118,11 +3119,16 @@ const definitions: Definition[] = [
         meta: {battery: {voltageToPercentage: '3V_2850_3000'}},
         exposes: [e.device_temperature(), e.battery(), e.battery_voltage()],
         extend: [
+            quirkAddEndpointCluster({endpointID: 1, inputCluster: 'msTemperatureMeasurement'}),
+            quirkAddEndpointCluster({endpointID: 1, inputCluster: 'msRelativeHumidity'}),
+            quirkAddEndpointCluster({endpointID: 1, inputCluster: 'genAnalogInput'}),
+            quirkAddEndpointCluster({endpointID: 1, inputCluster: 'aqaraOpple'}),
             aqaraAirQuality(),
             aqaraVoc(),
-            temperature({endpointID: 1}),
-            humidity({endpointID: 1}),
+            temperature(),
+            humidity(),
             aqaraDisplayUnit(),
+            reconfigureReportingsOnDeviceAnnounce(),
```

Comment:
I wasn't able to differentiate a rejoin where reporting was still working, or a rejoin where the device has been disconnected for more than 1h, were reporting was missing.

It shouldn't hurt to reconfigure them in both cases though.

Reply:
Going to remove this one, it seems to forget eveytime power_outage_count increases, but there is no announce/join to be found in the logs :(

I'll need to come up with a better way to do this one.